deployer/conf/elasticsearch.yml

```diff
@@ -58,6 +58,7 @@ path:
 searchguard:
   authcz.admin_dn:
   - CN=system.admin,OU=OpenShift,O=Logging
+  cert.oid: '1.3.6.1.4.1.2312.17.100.2.1'
```

Comment:
you also need to be checking the value for this to be secure - it's not enough to see it has this oid.  You need to verify that the incoming cert.oid value as an extension == the value you have on the cert you were given.  Basically, that the other person has a cert signed where this extension value == your extension value.

Maybe I'm not seeing all the changes.